ManualTestsReport.md

```diff
@@ -0,0 +1,245 @@
+- Prepare an SFTP Server.
+- Make sure the client public key is added to the list of trusted keys in the server.
+- Generate a connectionDataFile with all **valid** parameters.
+- Remove the private key from the path specified in the connectionDataFile
+
+| Step # | Action                                                     | Expected                                                      |
+|--------|------------------------------------------------------------|---------------------------------------------------------------|
+| 1      | Call the .exe file with all the parameters (see example 1) | Console message states: (see log2)    |
+| 2      | In destination path                                        | No new file has been created |
+
+log2:
+```
+Error connecting (check the connectionDataFile content)
+Invalid Private Key:
+```
+
+# Scenario 4: Cannot establish connection due to incorrect public key
+
+**Preconditions**: 
+- Prepare an SFTP Server.
+- Make sure the client public key is added to the list of trusted keys in the server.
+- Generate a connectionDataFile with all **valid** parameters.
+- Edit the public key in the client.
+
+| Step # | Action                                                     | Expected                                                      |
+|--------|------------------------------------------------------------|---------------------------------------------------------------|
+| 1      | Call the .exe file with all the parameters (see example 1) | Console message states: (see log3)|
+| 2      | In destination path                                        | No new file has been created |
+
+log3:
+```
+Error connecting (check the connectionDataFile content)
+The provided public key is invalid:
+```
+
+# Scenario 5: Cannot establish connection due to incorrect private key
+
+**Preconditions**: 
+- Prepare an SFTP Server.
+- Make sure the client public key is added to the list of trusted keys in the server.
+- Generate a connectionDataFile with all **valid** parameters.
+- Edit the private key in the client.
+
+| Step # | Action                                                     | Expected                                                      |
+|--------|------------------------------------------------------------|---------------------------------------------------------------|
+| 1      | Call the .exe file with all the parameters (see example 1) | Console message states: (see log4)   |
+| 2      | In destination path                                        | No new file has been created |
+
+log4:
+```
+Error connecting (check the connectionDataFile content)
+Invalid Private Key:
+```
+
+# Scenario 6: Cannot establish connection due to incorrect username
+
+**Preconditions**: 
+- Prepare an SFTP Server.
+- Make sure the client public key is added to the list of trusted keys in the server.
+- Generate a connectionDataFile with all **valid** parameters.
+- Edit the username in the connectionDataFile.

```

Comment:
In this case is not the private key is not necessary?

Reply:
Potentially it is not necessary, because if the validation of the username do not pass, the private key is not checked.